modules/stats.php

```diff
@@ -371,7 +371,7 @@ function stats_reports_page() {
 		// No JS fallback message
 ?>
 <div class="wrap">
-	<h2><?php esc_html_e( 'Site Stats', 'jetpack'); ?> <a style="font-size:13px;" href="<?php echo esc_url( admin_url('admin.php?page=jetpack&configure=stats') ); ?>"><?php esc_html_e( 'Configure', 'jetpack'); ?></a></h2>
+	<h2><?php esc_html_e( 'Site Stats', 'jetpack'); ?> <?php if ( current_user_can( 'manage_options' ) ) { ?><a style="font-size:13px;" href="<?php echo esc_url( admin_url('admin.php?page=jetpack&configure=stats') ); ?>"><?php esc_html_e( 'Configure', 'jetpack'); ?></a><?php } ?></h2>
```

Comment:
I came here to say the same :) 
but while I'm here I'd like to request a minor change to use `:`/`endif;` instead of brackets here.  
Just a style preference I know, but IMO easier to read amongst the html.